server/middlewares/changePasswordRequest.js

```diff
@@ -0,0 +1,37 @@
+/**
+ * Middleware to check if body parameters is null
+ * @param {Object} req
+ * @param {Object} res
+ * @param {Object} next
+ *
+ * @returns {Request|Response|*|void|boolean} res
+ */
+export default function checkPasswordRequest(req, res, next) {
+  const errors = {};
+
+  if (req.body.oldPassword === undefined || req.body.oldPassword === null) {
```

Comment:
Can you have a function that checks if the value passed to it is null or undefined instead of doing a similar check in the same long way?

Reply:
You do this in a lot of the files, so you might want to extract it to a util file.